### ISSUE TITLE
opactity:0 can hide input from screenreaders

### ISSUE THREAD

**Issue by pkra** [mdo/wtf-forms#70]
E.g., ChromeVox will ignore elements with opacity:0 (no way to navigate to them);  other screenreaders might be affected as well (haven't had the time to check). Dimensions of 1px seem to work as an alternative.

**Comment by pkra** [comment]
Correction: a 1px hack does not help after all (not sure why I had jotted that down during testing).

**Comment by StommePoes** [comment]
We've used opacity: 0 as one of the go-to methods for hiding controls while allowing access because it worked so well for everything (including Dragon, which is quite picky). ChromeVox is the first I've heard, and I think Google should fix it on their end since the literature tends to only mention display: none and visibility: hidden as the CSS methods of hiding stuff from AT.

**Comment by pkra** [comment]
@StommePoes agreed. This was filed over two years ago and ChromeVox was rewritten completely since then so it might well be fixed.
